Hpr007 (#7)
halfpastviberecords/catalog: hpr007_same-reason/README.md

# HPR007: Mitchellitchell - Same Reason

**Release Date:** 2025.11.14

## About
Half Past Vibe Records is pleased to present Mitchellitchell's first release on HPR "Same Reason."

> Mitchell Rossit-Lavigne is a long time music creator -- most notable for his eclectic and dance oriented sets, he has often pushed the limits of the DJ space through the inclusion of live instrumentation and unreleased tracks. We here at Half Past Vibes are excited to be the first label to release anything by this talented artist.

Reviewer: ```suggestion
Mitchell Rossit-Lavigne is a long time music creator -- most notable for his eclectic and dance oriented sets, he has often pushed the limits of the DJ space through the inclusion of live instrumentation and unreleased tracks. We here at Half Past Vibe are excited to be the first label to release anything by this talented artist.
```

This release comes packed with Rossit-Lavigne's new track accompanied by two gorgeous remixes. The first comes straight from Germany via our long time collaborator Anodized. While the second release comes from another artist making their debut on HPR -- Okibi -- a long time collaborator of label head Dataist.

The future of HPR is looking very bright as we expect many more great releases from Mitchellitchell which are bound to increase the market share of the dancefloor to any DJ lucky enough to come across these tunes.

## Usage
We believe this offering of house tracks -- broad enough to cover all your bases while still maintaining a central danceability -- should provide the user the ability to satisfy the deepest of dancefloor desires.

> 1. `Same Reason (Original Mix)` is on one hand a letter of love to Madrid -- where Rossit-Lavigne current resides -- through its sultry rhythms and textures. On the other hand, this track is overflowing with modern electronic potency through a combination of clarity, depth, and beautify lush synthetic sounds. This track is best served with a bottle of wine and a street part -- but will provide excellent value at nearly any time in the club.

Reviewer: ```suggestion
1. `Same Reason (Original Mix)` is on one hand a letter of love to Madrid -- where Rossit-Lavigne current resides -- through its sultry rhythms and textures. On the other hand, this track is overflowing with modern electronic potency through a combination of clarity, depth, and beautiful lush synthetic sounds. This track is best served with a bottle of wine and a street party -- but will provide excellent value at nearly any time in the club.
```

2. `Same Reason (Anodized Remix)` in classic style, Anodized has once again found a way to temper a house track into something a little harder hitting all while keeping the vibe of the source material. This track is expected to function optimally when the lights are low and the dancers are ready to let it all go.

> 3. `Same Reason (Okibi Dub)` comes from the extremely eclectic nature of Okibi. A long time scratch DJ and producer of music from dubstep to footwork (and everything in-between), this dubbed out version comes lush with reverberance without losing the plot of danceability through its garage-esq sensibilities. This track will serve to calm the most hectic nerves, fill a moody space, or bridge the gaps between North American and UK sensibilities.

Reviewer: ```suggestion
3. `Same Reason (Okibi Dub)` comes from the extremely eclectic nature of Okibi. A long time scratch DJ and producer of music from dubstep to footwork (and everything in-between), this dubbed out version comes lush with reverberance without losing the plot of danceability through its garage-esque sensibilities. This track will serve to calm the most hectic nerves, fill a moody space, or bridge the gaps between North American and UK sensibilities.
```
